site/content/internal/release-process.md

```diff
@@ -6,10 +6,10 @@ section: internal
 
 # Release Cutting Process
 
-**Note: To cut a build you need access to matterbuild. Please ask Christopher Speller for access if you don't have it.**
-
 On code complete day, work with the PM on rotation to get all the pull requests for the current release merged into `master`. Once that is done and you've confirmed with the PM, cut the first release candidate by following these steps:
 
+1. Give yourself access to matterbuild by adding yourself in the platform-private repo. See an [example here](https://github.com/mattermost/platform-private/commit/89f91d81bd4602f4708270c0ca7626da8fc45291) of Chris Brown adding himself.
+1. Trigger the matterbuild Jenkins job to update it https://build.mattermost.com/job/matterbuild.
```

Comment:
Numbers are wrong.

Reply:
It's markdown, it doesn't matter

Reply:
I think source readability still matters, but if we want to create ordered lists without maintaining the number series, can we at least use "1." for every bullet?